src/core/src/core_logic/PatchInstaller.py

```diff
@@ -736,3 +781,6 @@ def filter_out_excluded_updates(self, included_packages, included_package_versio
         self.composite_logger.log_debug(str(len(new_included_packages)) + " out of " + str(len(included_packages)) + " packages will remain included in the run.")
         return new_included_packages, new_included_package_versions
     # endregion
+
+    def get_max_batch_size(self, maintenance_window, package_manager):
```

Comment:
Where is the definition?

Reply:
The definition will be added in later iterations as we discussed offline. Trying to limit the size of the PR so that it is easier to review.